Services/Movies/Movies.API/Entities/MoviePerson.cs

```diff
@@ -1,17 +1,20 @@
 ﻿using System.ComponentModel.DataAnnotations;
+using Movies.API.Constants;
 
 namespace Movies.API.Entities
 {
     public class MoviePerson
     {
-        [Required(ErrorMessage = "MovieId is required.")]
+        [Required(ErrorMessage = ErrorMessages.MOVIE_ID_REQUIRED)]
         public int MovieId { get; set; }
-        [Required(ErrorMessage = "RoleId is required.")]
+        public Movie? Movie { get; set; }
```

Comment:
Ova polja ne moraju da budu nullable, postojanje odgovarajuceg Id polja to garantuje